lib/datadog/notifications/version.rb

```diff
@@ -1,5 +1,5 @@
 module Datadog
   class Notifications
-    VERSION = '0.5.1'.freeze
+    VERSION = '0.6.0'.freeze
```

Comment:
0.5.2 is fine, no need for major change

Reply:
@dim well, yes, given this is `0.x.x`, but I followed: https://semver.org/

> MINOR version when you add functionality in a backwards-compatible manner, and
> PATCH version when you make backwards-compatible bug fixes.

Reply:
yes, there is no breaking change here. with versions <1 semver is different. MINOR version are for breaking changes